bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -306,6 +306,15 @@ def MakeNumpyDataFrameCopy(np_dict):
                 return MakeNumpyDataFrame(np_dict)
 
             ns.FromNumpy = MakeNumpyDataFrameCopy
+
+            # make a RDataFrame from a panda dataframe
+            def MakePandaDataFrame(df):
```

Comment:
It's called "pandas", so can we maybe call it `MakePandasDataFrame`?

Sorry for only chiming in now, I was unaware of the PR before it got merged.

Reply:
Apologies for not noticing and not adding you to the loop. Addressed here https://github.com/root-project/root/pull/14990

Reply:
The function is not visible, what is visible is `ROOT.RDF.FromPandas` which is consistent with `ROOT.RDF.FromNumpy`

Reply:
yes, indeed it's internal! I applied a minimal change in #14990 anyway it was very easy :)